packages/react-native-codegen/src/parsers/typescript/components/events.js

```diff
@@ -25,6 +25,7 @@ const {
   throwIfArgumentPropsAreNull,
 } = require('../../error-utils');
 const {getEventArgument} = require('../../parsers-commons');
+import {emitDoubleProp} from '../../parsers-primitives';
```

Comment:
unfortunately, codegen does not support this way of importing functions.
Could you please update this with this, please:
```suggestion
const {emitDoubleProp} = require('../../parsers-primitives');
```
🙏

Reply:
Done! I fixed for flow and forgot to do it for typescript.